records/new_metadata/newmeta_spec.yaml

```diff
@@ -57,6 +57,23 @@ pub:
   type: pub
   edge_annotation: true
 
+upstream:
+  description: Number of postsynaptic densities on a neuron in a specific region. Use this to annotate has_presynaptic_terminal_in relationships.
```

Comment:
'Number of postsynaptic densities **(PSDs)** **annotated** on a neuron in a specific region.'?

It's not strictly true that the number reported is the same as the number actually there, but maybe that's not helpful to point out?

'Use this to annotate has_**post**synaptic_terminal_in relationships.'

Unless I'm misunderstanding what the relationship is, should be postsynaptic no?

Reply:
Fixed. See latest commit.